### PR TITLE
test(spanner): configurable host for it tests

### PR DESCRIPTION
**Pull request by thiagotnunes**
Makes it possible to set a different spanner api host to be used for the integration tests.

### CODE REVIEW

**Review by mohanli-ml — APPROVED** [review]
LGTM. Thanks!

File: spanner/integration_test.go [hunk 1]
```diff
@@ -165,6 +169,10 @@ func parseInstanceName(inst string) (project, instance string, err error) {
 	return matches[1], matches[2], nil
 }
 
+func getSpannerHost() (string) {
```
Comment (olavloite) [inline spanner/integration_test.go:172]: This needs to be the following for the format check to stop complaining:

```suggestion
func getSpannerHost() string {
```

You can check the formatting by executing `gofmt -s .` in the `cloud.google.com/spanner` directory.

**Review by olavloite — APPROVED** [review]
LGTM, but needs a small change for the formatter to stop complaining.

**Comment by thiagotnunes** [comment]
@olavloite thanks for spotting that!

**Pull request merged**